src/AplosConnector.Common/Models/MappingSettingsModel.cs

```diff
@@ -75,5 +77,8 @@ public class MappingSettingsModel
         public ExpenseAccountMappingModel[] ExpenseAccountMappings { get; set; }
         public TagMappingModel[] TagMappings { get; set; }
         public FundingSource PEXFundingSource { get; set; }
+
+        public double? SyncTransactionsIntervalDays { get; set; }
```

Comment:
These sound more like `int` based on the name, but if they must have decimals, the `decimal` type is preferred

Reply:
they are doubles because we can do fractions of a day (which we had to do for one customer even!). decimal is for money, we don't need precision.

Reply:
I don't agree that decimal is just for money, and think we should use it, but I will approve it since it's consistent with other apps.